components/menuData.ts

```diff
@@ -977,12 +985,16 @@ menuDataRouter.get(
             },
         }
 
-        if (req.gameVersion === "h1") {
+        availableDiff: if (req.gameVersion === "h1") {
             const inventory = createInventory(
                 req.jwt.unique_name,
                 req.gameVersion,
             )
 
+            if (LOCATION === "LOCATION_PARENT_ICA_FACILITY") {
+                break availableDiff
+            }
```

Comment:
Why can't we just include this in the outer check? Breaking control flow here is just weird.

Reply:
Good point, fixed.